site/getstarted/install.md

```diff
@@ -17,10 +17,13 @@ libraryDependencies += "co.fs2" %% "fs2-io" % "<version>"
 libraryDependencies += "co.fs2" %% "fs2-reactive-streams" % "<version>"
 ```
 
-The fs2-core library is also supported on Scala.js:
+The fs2-core as well as fs2-io libraries are also supported on Scala.js:
 
 ```
 libraryDependencies += "co.fs2" %%% "fs2-core" % "<version>"
+
+// not that fs2-io for scala.js only supports the node environment
+libraryDependencies += "co.fs2" %%% "fs2-io" % "<version>"
```

Comment:
```suggestion
libraryDependencies += "co.fs2" %%% "fs2-core" % "<version>"
libraryDependencies += "co.fs2" %%% "fs2-io" % "<version>" // Node.js only
```

Reply:
can't believe I had a typo in a one-line comment commit 🤦‍♂️

Reply:
Happens to the best of us. Thanks for updating the docs! Hope to see more projects using this stuff :)

Reply:
Likewise! I think all the stuff you've been working on is very exciting, and I can't wait to see how feral evolves!